src/main/java/nl/avans/drivio/repository/FuelCarRepository.java

```diff
@@ -1,9 +1,10 @@
 package nl.avans.drivio.repository;
 
 import nl.avans.drivio.model.Car;

```

Comment:
Is het Carmodel hier nog nodig als import?

Reply:
Ik heb de wijziging uitgevoerd (zie commit [[Feat: Changed the requested changes from PR#25](https://github.com/leonvsc/Drivio/pull/25/commits/8fa43389fecb9ea25438afd652b9d161e75421ff))](https://github.com/leonvsc/Drivio/pull/25/commits/8fa43389fecb9ea25438afd652b9d161e75421ff)